src/bin.js

```diff
@@ -69,23 +69,10 @@ commander
 
 // Number Buy
 commander
-  .command('number:buy <number>')
-  .description('Buy a number to use for voice or SMS')
+  .command('number:buy [number_pattern]')
+  .description('Buy a number to use for voice or SMS. If a --country_id is provided then the number_pattern is used to search for a number in the given country.')
   .alias('nb')
-  .option('--confirm', 'skip confirmation step and directly buy the number' )
-  .on('--help', () => {
-    emitter.log('  Examples:');
-    emitter.log(' ');
-    emitter.log('    $ nexmo number:buy 445555555555');
-    emitter.log('    $ nexmo number:buy 31555555555');
-    emitter.log('    $ nexmo number:buy 17136738555');
-    emitter.log(' ');
-  })
-  .action(request.numberBuy.bind(request));
-
-commander
-  .command('numbers:buy [country_code] <number>', null, { noHelp: true })
-  .description('Buy a number to use for voice or SMS')
+  .option('-c, --country_id [country_id]', 'the country code')
```

Comment:
Agreed. Went with `country_id` since it was used here https://github.com/Nexmo/nexmo-cli/blob/e85ab7c2f17c6478d62412029f3381f7cb8c70ac/src/request.js#L40. But that's really an internal variable name.

Will update.